src/sdf_tools/sdf.cpp

```diff
@@ -613,6 +613,34 @@ namespace sdf_tools
         }
     }
 
+    std::pair<double, bool> SignedDistanceField::DistanceToBoundary(const double x, const double y, const double z) const
+    {
+        return DistanceToBoundary4d(Eigen::Vector4d(x, y, z, 1.0));
+    }
+
+    std::pair<double, bool> SignedDistanceField::DistanceToBoundary3d(const Eigen::Vector3d& location) const
+    {
+        return DistanceToBoundary4d(Eigen::Vector4d(location.x(), location.y(), location.z(), 1.0));
+    }
+
+    std::pair<double, bool> SignedDistanceField::DistanceToBoundary4d(const Eigen::Vector4d& location) const
+    {
```

Comment:
assert( abs( location(3) - 1.0 ) < eps)

Reply:
I'm going to move this one off to a separate issue so that it can be addressed in the entire library without making this merge about 2 things.